<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVConfigDialog</class>
 <widget class="QDialog" name="CSVConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import CSV</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="file_layout">
     <item>
      <widget class="QLabel" name="file_label">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_line_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="file_button">
       <property name="text">
        <string>select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="h_line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="infos_preset_layout">
     <item>
      <widget class="QLabel" name="preset_label">
       <property name="text">
        <string>Preset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="preset_cbox"/>
     </item>
     <item>
      <widget class="QPushButton" name="save_cfg_button">
       <property name="text">
        <string>Save configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="options_columns_hlayout">
     <item>
      <layout class="QVBoxLayout" name="options_vlayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="options_label">
           <property name="text">
            <string>Options:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="options_table"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="columns_vlayout">
       <item>
        <widget class="QLabel" name="columns_label">
         <property name="text">
          <string>Columns:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="columns_table"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="h_line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="preview_label">
     <property name="text">
      <string>Preview:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="preview_table">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSVConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSVConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
